rebar.config.script

```diff
@@ -87,13 +87,18 @@ MakeDep = fun
         {AppName, ".*", {git, Url, Version}, Options}
 end,
 
+ErlOpts = case os:getenv("ERL_OPTS") of
+    "bin_opt_info" -> [bin_opt_info, debug_info, {i, "../"}];
```

Comment:
If debug_info is provided already from rebar,  why add it again?

Would this work?

```
ErlOpts = case os:getenv("ERL_OPTS") of
    false -> [{i, "../"}];
    ErlOpt -> [ErlOpt, {i, "../"}]
end
```

Reply:
`debug_info` in clause is a copy-paste error, thanks for pointing.

Passing value of env variable not going to work though, options have to be erlang term i.e. `bin_opt_info` need to be an atom, not a string. Also just marshalling os env into options is kind of dangerous.

Reply:
Ah, you're right it is a string. Let's keep it as a special case